# Case Background
A Columbia researcher, Dr. Chan, is comparing students' academic motivation scores in two different high schools in New York City, one school has mandatory uniforms for students and the other one that has no such requirement.  

The main dependent variable is academic motivation score.  

The researcher is concerned that students in School A (with the uniforms policy) may differ in academic ability and social economic status from students in school B (without the uniforms policy).  

1. takes a sample of 7 students from each of the school A and B
2. creates pairs of 8th-grade students, one from each school, who are matched on both a reading score measure from a state test, and on a classification of the student’s social economic status into five ordered categories based on student responses to a questionnaire.  
3. Sample scores on the academic motivation measure from seven of these matched pairs of students are given below.  A higher score indicates more positive motivation.

A school students: 23, 21, 17, 27, 27, 18, 20

B school students: 23, 16, 20, 21, 23, 14, 21

# Problem 1
Analyze the data given above to test (at .05) if academic motivation differs at the two schools, controlling for academic ability and social economic status. 

In [1]:
from scipy import stats

In [2]:
# input data
data1=[23, 21, 17, 27, 27, 18, 20]
data2=[23, 16, 20, 21, 23, 14, 21]

In [3]:
stats.ttest_rel(data1,data2)

Ttest_relResult(statistic=1.6497800439902246, pvalue=0.15008123419913932)

This test is used when the samples are dependent; that is, when there is only one sample that has been tested twice (repeated measures) or when there are two samples that have been matched or "paired". This is an example of a paired difference test.

Examples for the use are scores of the same set of student in different exams, or repeated sampling from the same units. The test measures whether the average score differs significantly across samples (e.g. exams). If we observe a large p-value, for example greater than 0.05 or 0.1 then we cannot reject the null hypothesis of identical average scores. If the p-value is smaller than the threshold, e.g. 1%, 5% or 10%, then we reject the null hypothesis of equal averages. Small p-values are associated with large t-statistics.

Reference: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ttest_rel.html

# Answer for Problem 1: 
The p-value=0.15 larger than 0.05, fail to reject the null hypothesis. Academic motivation does not differ at the two schools.